massa-execution/src/interface_impl.rs

```diff
@@ -155,4 +158,22 @@ impl Interface for InterfaceImpl {
             .map(|addr| addr.to_bs58_check())
             .collect())
     }
+
+    fn generate_event(&self, data: String) -> Result<()> {
+        let slot = context_guard!(self).slot;
+        let bytes_address = *context_guard!(self).call_stack.back().unwrap(); // TODO what's an empty stack ?
```

Comment:
for now the stack should always be non-empty, but to be future-proof and robust, we did a clean fail in the other methods of InterfaceImpl. Let's do the same here